esmvaltool/cmorizers/data/formatters/datasets/cds_uerra.py

```diff
@@ -43,12 +43,12 @@
 import numpy as np
```

Comment:
This script does not work. I guess the call to the cmorizer function should be changed (this [line](https://github.com/ESMValGroup/ESMValTool/blob/b8a7dcc6d1e4d06649bb38d831007ee439ec5a16/esmvaltool/cmorizers/data/formatters/datasets/cds_uerra.py#L248))